src/middlewared/middlewared/plugins/filesystem_/acl_template.py

```diff
@@ -54,10 +61,14 @@ async def validate_acl(self, data, schema, verrors):
         "acltemplate_create",
         Str("name", required=True),
         Str("acltype", required=True, enum=["NFS4", "POSIX1E"]),
-        List("acl", items=[Ref("posix1e_ace"), Ref("nfs4_ace")], required=True),
+        OROperator(Ref('nfs4_acl'), Ref('posix1e_acl'), name='acl', requried=True),
         register=True
     ))
-    async def create(self, data):
+    @returns(Ref('acltemplate_create'))
```

Comment:
The returned object would mirror `acltemplate_entry` and not `acltemplate_create` ? If i am accurate, we don't need to add an explicit `returns` as for CRUD methods, middleware will automatically do that for us based on `ENTRY` specification